api_v3/services/MediaService.php

```diff
@@ -149,6 +149,16 @@ protected function replaceResource(KalturaResource $resource, entry $dbEntry, $c
 		}
 		else
 		{
+			$kResource = $resource->toObject();
+			if ( $kResource instanceof kOperationResource ) {
+				$internalResource = $kResource->getResource();
+				if ( $dbEntry->getIsTrimDisabled()
```

Comment:
Although currently the only implementation of kOperationAttributes is kClipAttributes, I still would check it in case we will add more implementations in the future.